src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -461,16 +480,42 @@
     }
   }
 
+  .topRowWrapper {
+    display: flex;
+    flex-direction: row;
+    gap: min(var(--space-lg), 50px);
+    position: sticky;
```

Comment:
I think we should aim to not have to scroll this screen at all (so this being sticky shouldn't be needed). Scrolling is kinda annoying on the Steam Deck and when we have scroll in the whole screen it means the `play` button can start outside of the viewport when it makes sense for this button to always be visible since it's kinda the main action we want.

I think at some point I had that implemented (the other components would shrink if needed) but I can't see it working that way in the main branch either so I don't know what happened there.

If something like system requirements or game description are long we can have scroll on those elements that are not that critical for example.

Reply:
Main info element was *partially* implemented like this, so description would become scrollable, but `max-height` was too high and whole screen scrolled anyway. I will try to fix this.

Reply:
Should be fixed now. Now gamepage should almost always adapt to full screen and not be scrollable. One side note that under 600px gamepage *will* become scrollable since scrollable game description is not sufficient (for preventing overflow) anymore.